src/aat/resources/features/F-056/F-056.feature

```diff
@@ -52,4 +52,18 @@ Feature: F-056: Submit event creation as a Citizen
     And the response [includes the case detail for the updated case, along with a HTTP 200 OK]
     And the response has all other details as expected
 
+  @S-582
+  Scenario: must update successfully the respective fields with ACL permissions for a Citizen
+    Given a user with [an active Citizen profile in CCD with a specific variation of ACLs on a case type]
+    And a successful call [to create a token for case creation] as in [S-582_Citizen_Token_Creation_Data]
+    And another successful call [by a privileged user with full ACL to create a case of this case type] as in [S-582_Citizen_Case_Creation_Data]
```

Comment:
this can now reference the privileged user mentioned above, eg.

```
And another successful call [by the privileged user to create a case of this case type] as in [S-582_Citizen_Case_Creation_Data]
```